### PR TITLE
fix: fix value_counts column label for normalize=True

### PR DESCRIPTION
**Pull request by TrevorBergeron**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: tests/system/small/test_dataframe.py [hunk 1]
```diff
@@ -3385,6 +3385,8 @@ def test_df_to_orc(scalars_df_index, scalars_pandas_df_index):
     ],
 )
 def test_df_value_counts(scalars_dfs, subset, normalize, ascending, dropna):
+    if pd.__version__.startswith("1."):
+        pytest.skip("pandas 1.x produces different column labels.")
```
Comment (shobsi) [inline tests/system/small/test_dataframe.py:3389]: insert update expected value and assert?

Comment (TrevorBergeron) [reply tests/system/small/test_dataframe.py:3389]: I don't think its worth the tests complexity - we're validating against 2.x behavior and I think that is enough.

File: tests/system/small/test_dataframe.py [hunk 2]
```diff
@@ -3396,10 +3398,6 @@ def test_df_value_counts(scalars_dfs, subset, normalize, ascending, dropna):
         subset, normalize=normalize, ascending=ascending, dropna=dropna
     )
 
-    # Older pandas version may not have these values, bigframes tries to emulate 2.0+
```
Comment (shobsi) [inline tests/system/small/test_dataframe.py:3399]: Let's remove this from test_series as well https://github.com/search?q=repo%3Agoogleapis%2Fpython-bigquery-dataframes%20%22Older%20pandas%20version%20may%20not%20have%20these%20values%22&type=code

Comment (TrevorBergeron) [reply tests/system/small/test_dataframe.py:3399]: done

Comment (shobsi) [reply tests/system/small/test_dataframe.py:3399]: Thanks, just one one more test `test_value_counts_w_cut` in test_series.py

Comment (TrevorBergeron) [reply tests/system/small/test_dataframe.py:3399]: done

**Review by shobsi — APPROVED** [review]

**Review by shobsi — APPROVED** [review]

**Pull request merged**